ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4763,4 +4763,21 @@ kong_apis:
       config.hour: "{{ medium_rate_limit_per_hour }}"
       config.limit_by: credential
     - name: request-size-limiting
-      config.allowed_payload_size: "{{ small_request_size_limit }}"
\ No newline at end of file
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: listForm
+      uris: "{{ data_service_prefix }}/v1/form/list"
+      upstream_url: "{{ player_service_url }}/plugin/v1/form/list"
+      strip_uri: true
+      plugins:
+      - name: cors
```

Comment:
JWT Plugin is missing, refer other APIs like userCreate for an example.

Reply:
Done